python/api/onnxruntime-python-api.py

```diff
@@ -0,0 +1,117 @@
+# A set of code samples showing different usage of the ONNX Runtime Python API
+import numpy as np
+import torch
+import onnxruntime
+
+MODEL_FILE = '.model.onnx'
+DEVICE_NAME = 'cpu' # Replace this with 'cuda' or equivalent for other devices
```

Comment:
Using 'cuda' would be a better example since CPU to CPU copies done inside ```run_with_data_copied_to_device``` can confuse users to think they're required to do so.

Reply:
I guess I was hoping to demonstrate the functionality without users having to install CUDA. I can change it to CUDA though.

Reply:
Will set CUDA if it is available